css-basics/css-basics_pseudo-park/css/styles.css

```diff
@@ -22,13 +22,15 @@
 
 .animal {
   padding: 16px;
-
   /*
-    The animals are very small.
-    The size of the emojis is controlled via the font size.
-    Set the font size to 32px.
+  The animals are very small.
+  The size of the emojis is controlled via the font size.
+  Set the font size to 32px.
   */
 }
+.animal {
+  font-size: 2rem;
```

Comment:
font-size 2rem ?

Reply:
thx Adrian. here you go: 

🦒

Reply:
2rem equals 16px. You can redefine a rem in your css. Default size is 8px. rem is easier to handle especially when useing font-size.